app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+    }, {
+      who: ["hesa"],
+      what: "requires this to enable “to create the student's DfE record”"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Qualification - subject",
+  items: {
+    question: "Subject of degree",
+    required: "First degree: Yes
+      Additional degree(s): No",
+    api: {
+      param: "qualifications.degrees[].subject",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "wants to see that a candidate can demonstrate subject and curriculum knowledge"
+    }, {
+      who: ["hesa"],
+      what: "requires this to “provide data on the qualifications of teachers.”"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Qualification - grade",
+  items: {
+    question: "Enter your degree grade",
+    required: "First degree: Yes
+      Additional degree(s): No",
+    api: {
+      param: "qualifications.degrees[].grade",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["hesa"],
+      what: "requires this to “provide data on the qualifications of teachers.”"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"],
+    notes: ["Not all postgraduate degrees have grades"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Qualification - year",
+  items: {
+    question: "What year did you graduate?",
+    required: "First degree: Yes
+      Additional degree(s): No",
+    api: {
+      param: "qualifications.degrees[].award_year",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["hesa"],
+      what: "requires this to “create the student's DfE record.”"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Qualification - institution",
+  items: {
+    question: "Institution where you studied",
+    required: "First degree: Yes
+      Additional degree(s): No",
+    api: {
+      param: "qualifications.degrees[].institution_details",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "may be interested in calibre of the institution studied at"
```

Comment:
A dance provider mentioned they'd use this information to make sure they didn't send a trainee teacher to one of the schools they had previously trained at.

Reply:
Added this note.